packages/app/obojobo-express/server/migrations/20221214210826-modify-users-add-last-login-column.js

```diff
@@ -0,0 +1,31 @@
+'use strict'
```

Comment:
I would probably regenerate these migrations so their filenames indicate a more recent date, to make sure they're run in the correct order.